pyproject.toml

```diff
@@ -7,10 +7,14 @@ license = "MIT"
 readme = "README.md"
 homepage = "https://github.com/iMicknl/python-tahoma-api"
 repository = "https://github.com/iMicknl/python-tahoma-api"
+packages = [
+    { include = "tahoma_api" },
+    { include = "tahoma_api/**/*.py" },
+]
 
 [tool.poetry.dependencies]
 python = ">=3.6"
-aiohttp = "^3.6.2"
+aiohttp = "3.6.1"
```

Comment:
Why downgrade the version?

Reply:
Currently I am loading the integration in ha-tahoma via manifest.json. In development, this conflicts with the built-in version of aihttp in HA. I downgraded it temporarily for development.

```
  "requirements": [
    "git+https://github.com/iMicknl/python-tahoma-api.git@enhancement/reuse_session#tahoma_api===0.1.0"
  ],
```